packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -80,7 +80,46 @@ function preload(baseModule: () => Promise<{}>, deps?: string[]) {
     })
   ).then(() => baseModule())
 }
-
+function ParseCjsDynamicImports(code: string) {
```

Comment:
I assume you're coming from another language like C# 🤔 
In JS/TS we mostly use `camelCase` for functions and variables. Could you change the names so it adopt the project style?

Reply:
sure, I should change to lowerCamelCase?
ParseCjsDynamicImports =>parseCjsDynamicImports?

Reply:
```
parseCjsDynamicImports
imports
codeArray
currentCode
codeEndIndex
dynamicImportMatch
```

Reply:
I'm sorry for my bad code style:joy:

Reply:
It is **NOT** bad code style! It's just not matching the common use JS code style.